src/assets/assets.md

```diff
@@ -22,6 +28,98 @@ owned by an account and can be sent to other users.
 * TOC
 {:toc}
 
+## Asset Model
+
+All assets have the following fields along with the additional fields that are
+specific to its category.
+
+**Required Fields**
+
+| Field       | Type        | Description                                        |
+|:------------|:------------|:---------------------------------------------------|
+| id          | String      | The asset's unique identifier.                     |
+| accountId   | String      | The asset's owning Centrapay account id.           |
+| category    | String      | Asset category ("money", "giftcard", or "token").  |
+| type        | String      | Asset type id used by payment option asset types.  |
+| liveness    | String      | Either "main" (live payments allowed) or "test".   |
+| description | String      | Displayable asset description.                     |
+| createdAt   | Date String | Date when the asset was created or issued.         |
+| status      | String      | "active" if the asset can be used for payments.    |
+
+
+## Money
+
+Money assets, being backed by real currency, are the most flexible asset types.
+Money is accepted for most payment requests, can be sent in arbitrary amounts
+and does not expire.
+
+Money assets have the following fields along with the base asset fields.
+
+**Required Fields**
+
+| Field    | Type           | Description                                                          |
+|:---------|:---------------|:---------------------------------------------------------------------|
+| currency | String         | Currency code, eg "NZD"                                              |
+| balance  | Numeric String | Current balance in the currency's smallest denomination (ie. cents). |
+
+
+## Gift Cards
+
+Gift cards are similar to money but have greater spending restrictions and are
+not always backed by real currency. Gift cards usually have an expiry date, are
+typically tied to a small number of merchants, and can only be sent in their
+entirety.
+
+Gift cards have the following fields along with the base asset fields.
+
+**Required Fields**
+
+| Field          | Type           | Description                                                          |
+|:---------------|:---------------|:---------------------------------------------------------------------|
+| issuer         | String         | The identifier for the issuer of the gift card.                      |
+| currency       | String         | Currency code, eg "NZD"                                              |
+| balance        | Numeric String | Current balance in the currency's smallest denomination (ie. cents). |
+| initialBalance | Numeric String | The balance when the asset was created.                              |
+
+**Optional Fields**
```

Comment:
Ditto, it'd be nice to have different terms for returned fields.

How about "Optional response fields"